oceanspy/utils.py

```diff
@@ -924,7 +924,7 @@ def to_180(x):
     return x + (-1) * (x // 180) * 360
 
 
-def get_combination(lst, select):
+def get_combination(lst, select):  # pragma: no cover
```

Comment:
I think we can get rid of this function. This is not used anywhere and does the same thing as itertools

Reply:
Yeah, I was going to ask you about that. I checked the logs and you had made it in 2022. I decided to remove coverage for that and some other (compilable) funcs

Reply:
removed